spec/paperclip/has_attached_file_spec.rb

```diff
@@ -38,6 +38,14 @@
       assert_adding_attachment('avatar').defines_callback('after_commit')
     end
 
+    it 'defines an after_commit callback if the class allows after_commit callbacks' do
+      assert_adding_attachment('avatar').defines_callback('after_commit')
+    end
+
+    it 'defines an after_destroy callback if the class does not allow after_commit callbacks' do
+      assert_adding_attachment('avatar', :unstub_methods => [:after_commit]).defines_callback('after_destroy')
```

Comment:
Line is too long. [110/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.